WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -618,16 +618,19 @@ public static ConstructionState AddCoordinationFee(Round round, ConstructionStat
 
 		round.LogInfo($"Available coordination: {availableCoordinationFee}.");
 
-		if (availableCoordinationFee > round.Parameters.AllowedOutputAmounts.Min)
-		{
-			coinjoin = coinjoin.AddOutput(new TxOut(availableCoordinationFee, coordinatorScriptPubKey))
-				.AsPayingForSharedOverhead();
-		}
-		else
+		var minEconomicalOutput = round.Parameters.MiningFeeRate.GetFee(coordinatorScriptPubKey.EstimateInputVsize() + coordinatorScriptPubKey.EstimateOutputVsize());
```

Comment:
This is problematic because we are including the cost of spending the generated output at current fee, what is obviously wrong. The output will be spent **in the future** and we don't don't what the fee rate will be at spending time. 

This change would make most coordinator outputs uneconomical during high fee rate times.

Reply:
I used 1 s/vb.